### PR TITLE
Autoplay

### PR DESCRIPTION
**Pull request by Nublaglasiem**
Fixed due to comments

### CODE REVIEW

File: turn/turn.py [hunk 1]
```diff
@@ -12,25 +11,56 @@ class Turn:
...
+        """Incoming actions"""
+        print('Incoming actions: ', end='')
+        for inc_act in self.player.incoming_actions:
+            if self.player.incoming_actions.index(inc_act) != len(self.player.incoming_actions) - 1:
+                print(inc_act.name, end=', ')
+            else:
+                print(inc_act.name, end='')
+        print()
+
+        """TODO: blocking"""
```
Comment (sorokinal) [inline turn/turn.py]: blocking это фаза защиты? 
Тогда после неё и перед очисткой входящих экшонов над впилить ещё тудуху на срабатывание входящих экшонов.

File: turn/turn.py [hunk 2]
```diff
@@ -12,25 +11,56 @@ class Turn:
...
-            action.type = action_data['action_type']
-        action.goal = action_data['goal']
-        return action
+    def define_action(self, action_data):
+        for action in self.available_actions:
+            self.available_actions.remove(action)
+            if not action.type:
+                action.type = action_data['action_type']
+            action.goal = action_data['goal']
+            return action
```
Comment (sorokinal) [inline turn/turn.py]: и выпили папку .idea из ПРки

**Review by sorokinal — DISMISSED** [review]

File: turn/turn.py (hunk 1 above)
Comment (Nublaglasiem) [reply turn/turn.py]: Естественно.

File: turn/turn.py (hunk 2 above)
Comment (Nublaglasiem) [reply turn/turn.py]: Готово.

**Review by Nublaglasiem — COMMENTED** [review]
Made inputing goal till character found
Clear incoming actions after blocking

**Review by Nublaglasiem — COMMENTED** [review]
Delete input_attacks and outcoming_actions, only incoming_actions left.
Action type checking.
Fixing hits to current_hits.
Fixing show incoming_actions.

**Review by Nublaglasiem — COMMENTED** [review]
Creating current_hits and max_hits.
3 Actions in Turn restored.
Minor ui changes.

**Review by Nublaglasiem — COMMENTED** [review]
Deleting .idea

**Review by Nublaglasiem — COMMENTED** [review]
incoming_actions added.
Input_action_data changed.
Ui, usability changes.
Added print('Round') to game.
Added name to Action.
ActionType value´s changed.
Turn: shows name, hits, incoming actions and available actions.
DocStrings

**Review by sorokinal — APPROVED** [review]

**Pull request merged**